.ci/build_kubernetes.sh

```diff
@@ -47,11 +47,24 @@ docker image pull ${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG} || (
     docker image push ${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG}
 )
 
+## Docker image for application elasticsearch container
+docker image pull ${DOCKER_USERNAME}/elasticsearch:${DOCKER_IMAGE_TAG} || (
```

Comment:
are you sure? Will we change the Dockerfile regularly somehow?

Reply:
What are you suggesting? Build image one time and hardcode reference here? 
I don't have experience with it and don't know about the consequences.

Reply:
We set it to static tag instead of commit.